acceptance_tests/features/pages/reporting_unit.py

```diff
@@ -12,3 +12,27 @@ def get_ru_details():
         "ru_ref": browser.find_by_id('RU_REF').text
     }
     return ru_details
+
+
+def get_associated_surveys():
+    surveys = []
+    survey_titles = browser.find_by_css('h3')
```

Comment:
We probably shouldn't be finding by css instead as it might change in the future